notebooks/quickstart/data-scientist/02-explore-dataset.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Dataset doesn't have a shape. Instead Dataset is a collection of assets.</p><p>Each asset has a shape and type attached. Asset are analogous to columns, its just that each asset can have of different shape.</p><p>e.g. A lung cancer dataset can have train_imgs, train_labels, test_imgs, test_labels as individual assets.</p><p> </p><p>Similarly in case of Family Age Dataset, there is one asset present <code>Age</code>,which represents the age of different people in a family.</p><p> </p><p>ages = family_age_dataset["age"]</p><p> </p><p>Now you can call,</p><p> </p><p>ages.public_shape</p><p> </p><p>ages.public_dtype</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7045/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>